### DISCUSSION TITLE
FluentDataGrid Virtualize OverscanCount

### DISCUSSION THREAD

**Discussion by nvangnoob** [microsoft/fluentui-blazor#2627] [Q&A] [answered]
How come FluentDataGrid does not have a OverscanCount when virtualize is true?  I would like to change it to a different value instead of the default 3.

**Comment by vnbaaij** [comment, marked as answer]
I found that missing as well the other day. So, I've added it about a week ago in a branch I'm working on 😀. Will be in release 4.11
